README.md

```diff
@@ -53,10 +53,10 @@ Here you can find the documentation for different versions:
 - Latest stable version: [GitHub docs](https://github.com/flame-engine/flame/tree/master-v0.x/doc)
 - Latest v1.0.0 version: [GitHub docs](https://github.com/flame-engine/flame/tree/1.0.0-rc8/doc)
 
-The complete documentation can be found [here](doc/README.md).
+The complete documentation can be found [here](https://github.com/flame-engine/flame/doc).
```

Comment:
404, needs branch or tag too :)

Reply:
Maybe we should link to the flame site docs? Assuming that will always be up to date with the latest released version.

Reply:
Duh! Fixed.

We mention the site a few sentences below